modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientInboundMessageHandler.java

```diff
@@ -385,6 +394,7 @@ private void writeError(long requestId, int opCode, Throwable err, ChannelHandle
             packer.packInt(ServerMessageType.RESPONSE);
             packer.packLong(requestId);
             writeFlags(packer, ctx);
+            packer.packLong(observableTimestamp(null));
```

Comment:
Does it make sense? It will just write the current timestamp, unrelated to the error itself

Reply:
We should write `now()` with every error, because we don't know if an operation has modified any data before it failed.

For example, a compute job can update one row and fail on another. As a user, I would expect to see the first update from the client anyway, so we need a recent timestamp.